packages/babel-helpers/src/helpers.js

```diff
@@ -1973,3 +1973,122 @@ helpers.wrapRegExp = helper("7.2.6")`
     return _wrapRegExp.apply(this, arguments);
   }
 `;
+helpers.slice = helper("7.6.0")`
+  import isNativeFunction from 'isNativeFunction';
+  function isString(value) {
+    return typeof value === "string" || (
+        typeof value === "object" && Object.prototype.toString.call(value) === "[object String]"
+    )
+  }
+
+  function toLength(value) {
+    var len = toInteger(value);
+    if (len < 0) {
+      return 0;
+    }
+    return Math.min(len, 9007199254740991);
+  }
+
+  function toInteger(value) {
+    const number = +value;
+    if (isNaN(number)) {
+      return 0;
+    }
+    if (number === 0 || number === Infinity || number === -Infinity) {
```

Comment:
Nit: none of these are necessary, the flooring won't affect them.

Reply:
Yeah I copy pasted from ES specification without really thinking.